### PR TITLE
198 long loading user info

### PR DESCRIPTION
**Pull request by KaterynaHerasymiva**
198: User info in header (messages/notification, photo/initials, first/last name) and 'Add group class' button on Timetable aren't displayed after authorization, only after refreshing the page

Change loading user info from DB to loading from localhost when sign-up

### CODE REVIEW

File: frontend/src/app/shared/components/header/header.component.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ export class HeaderComponent implements OnInit {
     currentUser: IUserShort;
 
     ngOnInit(): void {
-        this.authService.loadUser().subscribe();
+        this.authService.loadUser().subscribe(u => this.currentUser = u);
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/header/header.component.ts:20]: We have next line in this file for this

File: frontend/src/app/modules/auth/sign-up/sign-up.component.ts [hunk 2]
```diff
@@ -103,7 +103,8 @@ export class SignUpComponent extends BaseComponent implements OnInit {
         this.authService
             .signUp(this.email.value, this.password.value)
             .pipe(switchMap(() => this.createUser().pipe(this.untilThis)))
-            .subscribe(() => {
+            .subscribe((user) => {
+                localStorage.setItem('user', JSON.stringify(user));
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/auth/sign-up/sign-up.component.ts:107]: Are you sure model is the same?
And please, don't work with localStorage in components, only in services

File: frontend/src/app/core/services/auth.service.ts [hunk 3]
```diff
@@ -81,6 +81,12 @@ export class AuthService {
     }
 
     loadUser() {
+        const userFromStorage = localStorage.getItem('user');
+
+        if (userFromStorage) {
+            return of(JSON.parse(userFromStorage));
+        }
+
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/core/services/auth.service.ts:89]: No, no, no, please no
This function is used to update user, if you will make so, it won't be possible anymore

File: frontend/src/app/shared/components/header/header.component.ts (hunk 1 above)
Comment (KaterynaHerasymiva) [reply frontend/src/app/shared/components/header/header.component.ts]: fixed

File: frontend/src/app/modules/auth/sign-up/sign-up.component.ts (hunk 2 above)
Comment (KaterynaHerasymiva) [reply frontend/src/app/modules/auth/sign-up/sign-up.component.ts]: fixed

File: frontend/src/app/core/services/auth.service.ts (hunk 3 above)
Comment (KaterynaHerasymiva) [reply frontend/src/app/core/services/auth.service.ts]: fixed

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by NadiaMaksymchuk — APPROVED** [review]

**Pull request merged**